convert2rhel/unit_tests/checks_test.py

```diff
@@ -271,8 +271,8 @@ def test_convert2rhel_latest_log_check_env(caplog, monkeypatch, convert2rhel_lat
 
     local_version, dummy_ = convert2rhel_latest_version_test
     log_msg = (
-        "You are currently running %s and the latest version of convert2rhel is 0.22.\n"
-        "'CONVERT2RHEL_UNSUPPORTED_VERSION' environment detected, continuing conversion" % (local_version)
+        "You are currently running %s and the latest version of Convert2RHEL is 0.22.\n"
+        "'CONVERT2RHEL_UNSUPPORTED_VERSION' environment variable detected, continuing conversion" % local_version
```

Comment:
I'd like to expand the unit test as well in the parametrize to include versions with different major versions as well.

E.g.
`local_version` of `0.18`, `package_version` of `1.1`
`local_version` of `1.1`, `package_version` of `0.25`

Reply:
yeah I can do that! I'll add the changes in and get it pushed on the next commit!